content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -798,8 +799,6 @@ Each feature gate is designed for enabling/disabling a specific feature:
   to `IngressClass.spec.parameters`.
 - `Initializers`: Allow asynchronous coordination of object creation using the
   Initializers admission plugin.
-- `IPv6DualStack`: Enable [dual stack](/docs/concepts/services-networking/dual-stack/)
-  support for IPv6.
```

Comment:
Please leave this description in place. We don't remove feature descriptions when they graduate.